src/lib/plugins/clear.js

```diff
@@ -0,0 +1,172 @@
+// params format:
+// 0: initial match
+// 1: ign
+// 2: block
+// 3: number of blocks to remove
+let Item
+let mcData
+
+module.exports.server = (serv, { version }) => {
+  serv.commands.add({
+    base: 'clear',
+    info: "Clear a player's inventory.",
+    usage: '/clear <player> <item> <maxCount>',
+    parse: /^(\w+)?(?: (\w+))?(?: (\d+))?$/, // ex: /clear USERNAME_ stone 1
+    action (params, ctx) {
+      if (typeof params === 'string') { // the parsing failed so it just returned initial input
+        return `${this.usage}: ${this.info}`
+      }
+
+      Item = require('prismarine-item')(version)
+      mcData = require('minecraft-data')(version)
```

Comment:
Item and mcData should be loaded outside the command actions + there is no need for them to be in global scope.

Reply:
fixed